example/route/route.cpp

```diff
@@ -0,0 +1,144 @@
+//
+// Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CppAlliance/url
+//
+
+#include <boost/url/error.hpp>
+#include <boost/url/segments_encoded.hpp>
+#include <boost/url/segments_encoded_view.hpp>
+#include <boost/url/static_pool.hpp>
+#include <boost/url/string_view.hpp>
+#include <boost/url/url_view.hpp>
+#include <boost/url/static_url.hpp>
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/path.hpp>
+#include <iostream>
+
+namespace urls = boost::urls;
+namespace fs = boost::filesystem;
+using string_view = urls::string_view;
+
+class route
+{
+public:
+    route(string_view prefix, fs::path root)
+        : prefix_(urls::parse_uri_reference(prefix).value())
+        , root_(std::move(root))
+    {}
+
+    route(urls::url prefix, fs::path root)
+        : prefix_(std::move(prefix))
+        , root_(std::move(root))
+    {}
+
+    bool match(
+        urls::url_view target,
+        fs::path& result )
+    {
+        // Target segments
+        urls::static_pool<1024> pool;
```

Comment:
unused variable?

Reply:
Oh... I didn't get a warning. I have to check this.